kolibri/plugins/learn/assets/src/composables/useLearningActivities.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Everything related to learning activities of a content node
+ * and their duration.
+ */
+
+import { computed } from 'kolibri.lib.vueCompositionApi';
+import { get } from '@vueuse/core';
+import { LearningActivities } from 'kolibri.coreVue.vuex.constants';
+import { coreStrings } from '../../../../../core/assets/src/mixins/commonCoreStrings';
+
+const ReferenceLabel = coreStrings.$tr('readReference');
+const _LearningActivityToLabelMap = {
+  [LearningActivities.CREATE]: coreStrings.$tr('create'),
```

Comment:
Out of an abundance of caution, if we could move this logic inside the function scope that would be preferable, as we've previously had issues caused by accessing translations in module scope: https://github.com/learningequality/kolibri/issues/6003

(note - these only show up when the Intl polyfill is used, so it's hard to spot, and would only really occur in Safari).